### PR TITLE
Rebrand VM to Nextcloud Community

### LINKED ISSUE #12: Rebrand the VM TOTALLY
Right now there are a lot of references to Tech and Me, as I developed it there from start. I would appreciate a PR that changed the branding to "Nextcloud Community" instead.

Welcome with your PR! :)

### PR DESCRIPTION
**Pull request by mar1u5**
fixes #12, feedback welcome 😉 

cc @enoch85

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -78,14 +78,13 @@
 <div class="logo">
 </div>
 <div class="logotext">
-<h2>Nextcloud VM - <a href="https://www.techandme.se/nextcloud-vm/" target="_blank">Tech and Me</a></h2>
+<h2><a href="https://nextcloud.com/" target="_blank">Nextcloud VM</a></h2>
```
Comment (enoch85) [inline index.php]: Please link to the GitHub repo instead: https://github.com/nextcloud/vm

File: static/instruction.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 clear
 cat << INST1
 +-----------------------------------------------------------------------+
-| Thank you for downloading this Nextcloud VM made by Tech and Me!      |
+| Thank you for downloading this Nextcloud VM made by the Nextcloud Community!      |
```
Comment (enoch85) [inline static/instruction.sh]: Have you run the scripts so that you cheked that the text won't be off and symetric to the box?

**Comment by enoch85** [comment]
Fix the comments, then :+1:

**Comment by mar1u5** [comment]
Can you check my commits above 😉 @enoch85

**Comment by enoch85** [comment]
Thanks! :D Can you squash the commits? After that I'll merge. :rocket:

**Comment by mar1u5** [comment]
Searching for this option, but i dont find it :grin: 

https://help.github.com/articles/about-pull-request-merge-squashing/

**Comment by mar1u5** [comment]
Merged squashed :wink: THX.

**Comment by enoch85** [comment]
Thank *you* :)

**Pull request merged**